### DISCUSSION TITLE
Non-uniform light pollution?

### DISCUSSION THREAD

**Discussion by CervidaeAxis** [Stellarium/stellarium#3886] [General]
I was wondering if there's a way for Stellarium to simulate differing levels of light pollution across the sky. Reason is I live on the far outskirts of Cleveland, OH next to Lake Erie, and this results in a Bortle index of 6-7 that visibly drops off to 3-4 over the lake. Is it possible to replicate this in Stellarium, or does the program only support a single uniform value?

![image](https://github.com/user-attachments/assets/a8fb8f6b-8ad6-48b5-b750-7c24650c2a14)

**Comment by 10110111** [comment]
Non-uniformity of light pollution hasn't been implemented yet.

**Reply by CervidaeAxis** [reply]
Understood, thank you
